common/src/commonTest/kotlin/com/github/mustafaozhan/ccc/common/repo/SettingsRepositoryTest.kt

```diff
@@ -79,12 +80,88 @@ class SettingsRepositoryTest {
     }
 
     @Test
-    fun lastReviewRequest() {
+    fun default_lastReviewRequest() {
         given(settings)
             .function(settings::getLong)
             .whenInvokedWith(matching { it == KEY_LAST_REVIEW_REQUEST }, any())
             .thenReturn(DEFAULT_LAST_REVIEW_REQUEST)
 
         assertEquals(DEFAULT_LAST_REVIEW_REQUEST, repository.lastReviewRequest)
     }
+
+    // setters
+    @Test
+    fun set_firstRun() {
+        given(settings)
+            .function(settings::putBoolean)
+            .whenInvokedWith(matching { it == KEY_FIRST_RUN }, any())
+            .thenReturn(Unit)
+
+        repository.firstRun = true
+
+        verify(settings)
+            .function(settings::putBoolean)
+            .with(matching { it == KEY_FIRST_RUN }, any())
```

Comment:
How come you're using `matching { it == KEY_FIRST_RUN }` over `eq(KEY_FIRST_RUN)` here?

```suggestion
            .with(eq(KEY_FIRST_RUN), any())
```

Is it a matter of preference, did you not know about `eq`, or is it not working as expected?

Reply:
Thank you for pointing out this it is definitely working and better syntax :) I will update it 👍